test/models/work_test.rb

```diff
@@ -0,0 +1,52 @@
+require "test_helper"
+
+describe Work do
+
+  it 'can be instantiated' do
+    (Work.new).must_be_kind_of Work
+  end
+
+  it "will have the required fields" do
+    work = works(:test_book)
+    [:category, :title, :creator, :publication_year, :description].each do |check|
+      expect(work).must_respond_to check
+    end
+  end
+
+  describe "relationships" do
+    # Your tests go here
+  end
+
+  describe "validations" do
+    let work = works(:test_book)
```

Comment:
This isn't how a `let` is written

```suggestion
    let (:work) { works(:test_book) }
```